### ISSUE TITLE
Tab completion broken in 0.2.6

### ISSUE THREAD

**Issue by george-b** [tarruda/zsh-autosuggestions#117]
As of commit 2461a98857c137cbc3c93d0fa93e56854007af89 when I attempt to use tab completion I get the following.

```zsh
$ cd _main_complete:199: command not found: expand-or-complete
_main_complete:199: command not found: modify
_main_complete:199: command not found: expand-or-complete
_main_complete:199: command not found: modify
```

**Comment by ericfreese** [comment]
Forgot a backslash. This should be fixed in v0.2.7.

**Comment by george-b** [comment]
Thanks, I can confirm tab completion in working again.
